### ISSUE TITLE
a place to connect people who want PR reviews and people who want to review PRs

### ISSUE THREAD

**Issue by lpmi-13** [remotehack/remotehack.github.io#2]
A lot of times, when working on personal projects, there is just one contributor, and so not anyone obvious to help out with code reviews. This is especially relevant for new programmers (or experienced ones completing a project in a new language/framework), who are interested in improving their real-world skills, which definitely include giving comments and receiving comments during code review.

The basic goal could be a website that lists all the PR's retrieved from the github search API with a certain label (review-requested/please-review/icanhazreview/etc), with links to the PR.

...in preliminary investigation, I was able to bring back a list of PR's with a given label, eg
```
https://api.github.com/search/issues?q=label:review+is:pr+state:open&sort=created&order=asc
```

and an example result here: https://github.com/npm/fstream-ignore/pull/5

...so users could potentially do this now without a new tool, if *both* the PR submitter and the hopeful PR reviewer know the same label to use/search for...unfortunately they might not. So this way a given label could be agreed upon and used for higher discoverability.

**Comment by lpmi-13** [comment]
still needs to be upgraded to grab the data at build time instead of run time, and also probably filtered for "bad words" (:tm:)

**Comment by jerrysam** [comment]
https://codereview.stackexchange.com/ is likely an inspiration for this idea!

**Comment by lpmi-13** [comment]
Yeah, something like that site, but with the ability to click through to actual repositories. Might not actually add that much value, but seemed like a fun thing to play around with
